src/test/commp.test.ts

```diff
@@ -0,0 +1,67 @@
+/* globals describe it */
+
+/**
+ * Basic tests for CommP utilities
+ */
+
+import { assert } from 'chai'
```

Comment:
These days, `node --test` also works quite well, with the internal test and assert modules